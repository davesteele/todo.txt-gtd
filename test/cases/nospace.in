# A
task @work
# B
Another task @work
