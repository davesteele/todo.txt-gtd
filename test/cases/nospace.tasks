
@work
task @work
Another task @work
