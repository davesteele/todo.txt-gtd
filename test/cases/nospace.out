# A
task @work +A

# B
Another task @work +B

# _None
#

